lib/github/api.nit

```diff
@@ -757,6 +757,9 @@ class PullRequest
 
 	# Changed files count.
 	var changed_files: Int is writable
+
+	# URL to patch file
+	var patch_url: nullable String is writable
```

Comment:
Because the merge of the config with all the rest makes nitrpg use cache for testing (which is not a bad thing) and don't pass the tests.